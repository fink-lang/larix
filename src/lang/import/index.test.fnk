{describe, it, expect, to_match_snapshot} = import '../../testing/jest'

{parse_expr} = import '../..'


describe:: 'import `foobar`', fn:
  it:: 'parses import', fn:
    expect::
      parse_expr:: 'import `./foobar`'
      to_match_snapshot
