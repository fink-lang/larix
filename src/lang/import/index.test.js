import {parse_expr} from '../..';


describe('import `foobar`', ()=> {
  it('parses import', ()=> {
    expect(
      parse_expr('import `./foobar`')
    ).toMatchSnapshot();
  });
});
