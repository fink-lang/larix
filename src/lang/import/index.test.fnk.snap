// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import \`foobar\` parses import 1`] = `
import (1:0-1:17)
  string \` (1:7-1:17)
    './foobar'
`;
